packages/openneuro-app/src/scripts/errors/errorBoundary.jsx

```diff
@@ -142,16 +143,17 @@ class ErrorBoundary extends React.Component {
   }
 
   render() {
+    const error = this.props.error || this.state.error
     const { supportModal, message } = this.state
     const { subject, description } = this.props
-    const error = this.props.error || this.state.error
     const { loading } = this.props
 
+    if (loading) {
+      return <Spinner text="Dataset is currently being updated." active />
```

Comment:
Since loading occurs when the dataset is simply being fetched, the text might be changed to something like, "Dataset is currently being fetched." or "Fetching dataset..."